### PR TITLE
Fix extended audio for devo12 w/ TYPE=dev enabled

### PR DESCRIPTION
**Pull request by PhracturedBlue**

### CODE REVIEW

File: src/extended_audio.c [hunk 1]
```diff
@@ -49,8 +49,15 @@ void AUDIO_Init() {
         printf("Voice: UART5 already initialized\n");
         return;
     }
+    else
```
Comment (PhracturedBlue) [inline src/extended_audio.c]: Lint Error:
```
52:  If/else bodies with multiple statements require braces  
```

File: src/extended_audio.c [hunk 2]
```diff
@@ -49,8 +49,15 @@ void AUDIO_Init() {
...
 #endif
-
+    {
+#if defined BUILDTYPE_DEV
+        printf("Voice: Dev mode enabled, no audio output\n");
+        return;
+#endif
+    }
+
```
Comment (PhracturedBlue) [inline src/extended_audio.c]: Lint Error:
```
60:  Line ends in whitespace.  Consider deleting these extra spaces.  
```

File: src/extended_audio.c [hunk 3]
```diff
@@ -142,6 +149,16 @@ int AUDIO_Play(u16 music) {
 }
 
 void AUDIO_SetVolume() {
+#if defined BUILDTYPE_DEV
+#if HAS_AUDIO_UART 
+    if (! Transmitter.audio_uart)
```
Comment (PhracturedBlue) [inline src/extended_audio.c]: Lint Error:
```
153:  Line ends in whitespace.  Consider deleting these extra spaces.  
154:  If/else bodies with multiple statements require braces  
154:  Extra space for operator !   
```

File: src/extended_audio.c [hunk 4]
```diff
@@ -189,6 +206,17 @@ void AUDIO_CheckQueue() {
 }
 
 int AUDIO_VoiceAvailable() {
+#if defined BUILDTYPE_DEV
+#if HAS_AUDIO_UART 
+    if (! Transmitter.audio_uart)
```
Comment (PhracturedBlue) [inline src/extended_audio.c]: Lint Error:
```
210:  Line ends in whitespace.  Consider deleting these extra spaces.  
211:  If/else bodies with multiple statements require braces  
211:  Extra space for operator !   
```

File: src/extended_audio.c [hunk 5]
```diff
@@ -189,6 +206,17 @@ void AUDIO_CheckQueue() {
...
+#endif
+    {
+        printf("Voice: Dev mode enabled, cannot set volume\n");
+        num_audio = 0; // Reset queue when audio not available
```
Comment (PhracturedBlue) [inline src/extended_audio.c]: Lint Error:
```
215:  At least two spaces is best between code and comments  
```

File: src/extended_audio.c [hunk 6]
```diff
@@ -49,7 +49,14 @@ void AUDIO_Init() {
         printf("Voice: UART5 already initialized\n");
         return;
     }
+    else
```
Comment (PhracturedBlue) [inline src/extended_audio.c]: Lint Error:
```
52:  If/else bodies with multiple statements require braces  
```

File: src/extended_audio.c [hunk 7]
```diff
@@ -142,6 +149,16 @@ int AUDIO_Play(u16 music) {
 }
 
 void AUDIO_SetVolume() {
+#if defined BUILDTYPE_DEV
+#if HAS_AUDIO_UART
+    if (!Transmitter.audio_uart)
```
Comment (PhracturedBlue) [inline src/extended_audio.c]: Lint Error:
```
154:  If/else bodies with multiple statements require braces  
```

File: src/extended_audio.c [hunk 8]
```diff
@@ -189,14 +206,25 @@ void AUDIO_CheckQueue() {
 }
 
 int AUDIO_VoiceAvailable() {
+#if defined BUILDTYPE_DEV
+#if HAS_AUDIO_UART
+    if (!Transmitter.audio_uart)
```
Comment (PhracturedBlue) [inline src/extended_audio.c]: Lint Error:
```
211:  If/else bodies with multiple statements require braces  
```

**Comment by TheRealMoeder** [comment]
I think https://github.com/DeviationTX/deviation/commit/0416d69ee6c390cc49ad9f86ab7929495f4b5c22 is probably more appropriate, as the test doesn't make much sense anymore

File: src/extended_audio.c (hunk 6 above)
Comment (PhracturedBlue) [inline src/extended_audio.c]: Lint Error:
```
52:  If/else bodies with multiple statements require braces  
```

File: src/extended_audio.c (hunk 7 above)
Comment (PhracturedBlue) [inline src/extended_audio.c]: Lint Error:
```
154:  If/else bodies with multiple statements require braces  
```

File: src/extended_audio.c (hunk 8 above)
Comment (PhracturedBlue) [inline src/extended_audio.c]: Lint Error:
```
211:  If/else bodies with multiple statements require braces  
```

**Pull request merged**